.husky/pre-commit

```diff
@@ -1 +1 @@
-git rev-parse -q --no-revs --verify MERGE_HEAD && echo "Merge mode - pre-commit skipped" || npx expo lint
\ No newline at end of file
+git rev-parse -q --no-revs --verify MERGE_HEAD && echo "Merge mode - pre-commit skipped" || npx eslint . --fix && npx expo lint
```

Comment:
for convinience the script `npx eslint . --fix && npx expo lint` can be moved to nom scripts (optional)